torch/csrc/jit/passes/onnx.cpp

```diff
@@ -82,6 +84,7 @@ void BlockToONNX(Block* old_block, Block* new_block, ::torch::onnx::OperatorExpo
   // Clone the node and add it to the new graph
   auto cloneNode = [&](Node * node) {
     auto n_ = ctx.block->appendNode(ctx.block->owningGraph()->createClone(node, envFn));
+    n_->setScopeFromString(node->scopeName());
```

Comment:
Why do we need the string functions and can't simply `n_->setScope(node->scope())`?